### ISSUE TITLE
XBee library

### ISSUE THREAD

**Issue by Diaoul** [xoseperez/xbee2mqtt#1]
I was looking for XBee + MQTT and here I am :)
I'm currently using no protocol at all for my XBee sensors and I'm starting to think that maybe I should.
ZCL does that but is really complex.

I've developed my own XBee python library because the current one was too much binded to the Serial module and I wanted something modular, well documented and tested. I'd be glad if you could give me your impressions on it. Also, I wanted python3 compatibility.
https://github.com/Diaoul/hachi

**Comment by MYeager1967** [comment]
Is this for Python 2 or 3? I'm looking to convert an XBee network to a more robust protocol and MQTT is already in use on the platform I'm using. Is this still supported???

**Comment by xoseperez** [comment]
No, I moved my XBee WSN to RFM69 a while ago. Cheaper, more penetration and easier to use.

**Comment by MYeager1967** [comment]
Does it support two way communication as well? I have a few devices that I am controlling as well as collecting data from. I'm working to interface an older, homebrew system into Home Assistant...

**Comment by xoseperez** [comment]
You can do two way communication with the RFM69 too.
